less/search.less

```diff
@@ -23,7 +23,7 @@
     .transition(color .15s linear);
   }
 
-  .twitter-typeahead:hover + .fa, .fa:hover {
+  .twitter-typeahead:hover + .fa, .fa:hover, .twitter-typeahead:hover + .fas, .fas:hover {
```

Comment:
Could completely replace the `.fa` versions - we don't need to support FA4 icons.

Alternatively we only ever use `<i>` in here for icons so could select on that instead.

Reply:
Won't dropping `.fa` support stop SiteBuilder from deploying the shims w/o upgrading if it wants to?

Reply:
Yeah, but we'd expect Sitebuilder to upgrade its own icons to FA5 as part of this - the shims are just for user content.

Reply:
Fair enough, I'll simplify this selector